packages/composer/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/composer/composer
+TERMUX_PKG_DESCRIPTION="Composer is a tool for dependency management in PHP."
+TERMUX_PKG_LICENSE=MIT
+TERMUX_PKG_VERSION=2
+TERMUX_PKG_DEPENDS="php"
+
+termux_step_make_install() {
+  if [ -f $TERMUX_PREFIX/composer ]; then
+  rm $PREFIX/bin/composer
+  else
+  curl -sS https://getcomposer.org/installer | php -- --install-dir=$TERMUX_PREFIX/bin --filename=composer
```

Comment:
1. Never get sources through that way. You have `TERMUX_PKG_SRCURL` for source URL.
2. Highly recommended to have integrity verification there.
3. Piping output to interpreter is VERY BAD practice, especially in packaging.

Reply:
1.Yes,for now composer does not need to extract .I fixed for piping remote output problem.
2.I changed 99% of build.sh for fixing.
Fake version of Composer pkg version is that I miss to check at their github.Fixed.
3.Termux CURL does not need for composer it wi install itself by using its setup file.
4.Piping is resolved by 1 and 2.It will not show.
5.$prefix is wrong bcuz of my stupid copying from test file.Fixed